### PR TITLE
Add get-payment-response for Giropay

### PR DESCRIPTION
**Pull request by mikhail-kuzmiankou-cko**

### CODE REVIEW

File: spec/components/schemas/Payments/PaymentResponseGiropaySource.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          - giropay
+      purpose:
+        type: string
+        maxLength: 27
+        description: Purpose
+        example: Set of 3 masks
+      bic:
+        type: string
+        maxLength: 11
+        description: BIC (8 or 11-digits)
```
Comment (kevin-reece-cko) [inline spec/components/schemas/Payments/PaymentResponseGiropaySource.yaml]: "8 or 11 characters", as "digits" implies numbers only

Comment (mikhail-kuzmiankou-cko) [reply spec/components/schemas/Payments/PaymentResponseGiropaySource.yaml]:
```suggestion
        description: BIC (8 or 11 characters)
```

File: spec/components/schemas/Payments/PaymentResponseGiropaySource.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    properties:
+      type:
+        type: string
+        description: The payment source type
+        enum:
+          - giropay
+      purpose:
+        type: string
+        maxLength: 27
+        description: Purpose
```
Comment (ben-foster-cko) [inline spec/components/schemas/Payments/PaymentResponseGiropaySource.yaml]: Please add a clearer description of the `purpose` field

File: spec/components/schemas/Payments/PaymentResponseGiropaySource.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        example: Set of 3 masks
+      bic:
+        type: string
+        maxLength: 11
+        description: BIC (8 or 11 characters)
```
Comment (ben-foster-cko) [inline spec/components/schemas/Payments/PaymentResponseGiropaySource.yaml]: please use unabbreviated term in description i.e. "Bank Identifier Code (BIC)"

File: spec/components/schemas/Payments/PaymentResponseGiropaySource.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        example: TESTDETT421
+      iban: 
+        type: string
+        maxLength: 34
+        description: IBAN without whitespaces
```
Comment (ben-foster-cko) [inline spec/components/schemas/Payments/PaymentResponseGiropaySource.yaml]: Please use unabbreviated term in description i.e. "International Bank Account Number (IBAN)"

File: spec/components/schemas/Payments/PaymentResponseGiropaySource.yaml (hunk 2 above)
Comment (mikhail-kuzmiankou-cko) [reply spec/components/schemas/Payments/PaymentResponseGiropaySource.yaml]:
```suggestion
        description: Purpose of the payment
```

**Pull request merged**